### ISSUE TITLE
Not able to achieve accuracy shown in paper/repo

### ISSUE THREAD

**Issue by udion** [ZhixiuYe/HSCRF-pytorch#5]
I tried running the program in an environment with pytorch=0.2.0 and python=2.7 multiple times using the command : `CUDA_VISIBLE_DEVICES=0 python train.py --char_lstm  --high_way`

The highest accuracy that I have gotten is as follows : 
![image](https://user-images.githubusercontent.com/18963158/40692877-3bdeeb1c-63ef-11e8-803f-058f87bda359.png)

Please tell me how you were able to achieve 91.xx?

**Comment by ZhixiuYe** [comment]
Do you use the same random seed `5703959`?
Or you can try other random seeds due to the grate impact of the randomness of initialization.

**Comment by udion** [comment]
I used the same random seed as given in the code. Can you please be explicit and confirm what was the best result that you got.

My understanding from the paper is for NER task on ConNLL-2003 it is: `f1 score of 91.38%`

right?

**Comment by ZhixiuYe** [comment]
Yes, in my python environment, random seed 5703959 can get a pretty good result, `f1 score of 91.53%`. 
I don't know whether the generation fo random number is related to the python environment. So I suggest that you can try other random seed.
